radxa-RockPi4(C): Add SPI documentation (#125)
Tow-Boot/Tow-Boot: boards/radxa-RockPi4/README.md

# Radxa ROCK Pi 4 model A/B

## Device-specific notes

According to the [upstream SPI documentation](https://wiki.radxa.com/Rockpi4/hardware/spi_flash),
the SPI flash is only populated on board revisions V1.4 and later.

> While the schematics denote `W25Q64FV` as used flash, all known instances came

Reviewer: ```suggestion
While the schematics denote `W25Q64FV` is used, all verified units came
```

The wording was a bit weird, I'm fine with other rewrites too.

Main thing to consider:

 - "as used flash" doesn't work, I think.
 - "all known instances" is weird for a physical item.

Author: That's the result of non-native English :sweat_smile: Thanks, I've squashed the commits of PR and your suggestion together.

Reviewer: I'm not a native english speaker either :).
